src/composables/useTransactions.ts

```diff
@@ -55,7 +55,8 @@ export type TransactionAction =
   | 'stake'
   | 'restake'
   | 'sync'
-  | 'userGaugeCheckpoint';
+  | 'userGaugeCheckpoint'
+  | 'sorFetchPools';
```

Comment:
I'd rather not do this, this is meant to be a list of transaction actions. I have a PR which makes formatting the error message for Sentry better, so hopefully this shouldn't be needed.

Reply:
That's fine, but as is fetch pool errors are not getting logged, at all. This needs to change.